draft-ietf-quic-transport.md

```diff
@@ -3832,6 +3874,41 @@ the range from 0xFE00 to 0xFFFF.
 {: #iana-error-table title="Initial QUIC Transport Error Codes Entries"}
 
 
+## EXTENSION Frame Types {#iana-extension-types}
+
+IANA \[SHALL add/has added] a registry for "QUIC EXTENSION Frame Types" under a
+"QUIC Protocol" heading.
+
+The "QUIC EXTENSION Frame Types" registry governs a 16-bit space.  Values are
+assigned using the Expert Review policy {{!RFC8126}}.  Registrations can be
+provisional or permanent.  Permanent registrations can only be made with the
+inclusion of a specification, as defined by the Specification Required policy
+{{!RFC8126}}.  Provisional registrations can be removed at the discretion of the
+assigned expert after 1 year unless they are renewed.  Provisional registrations
+do not require that a specification be referenced.
+
```

Comment:
The spec was cool until that point. But there is a need to allow for experiments "between consenting implementations". What are they going to do? Squat on unallocated values? Or do we want to have an explicit range of extension types reserved for experiments, say FF00 to FFFE?